pkg/plugin/polaris/plugin_test.go

```diff
@@ -9,8 +9,7 @@ import (
 	"github.com/aquasecurity/starboard/pkg/ext"
 	"github.com/aquasecurity/starboard/pkg/plugin/polaris"
 	"github.com/aquasecurity/starboard/pkg/starboard"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
+	"github.com/onsi/gomega"
```

Comment:
On the mission to replace Testify with Gomega matchers, which are more powerful and we use them in integration tests.

Reply:
I get the reasoning but I personally would still vote for testify just because I find them simpler to write. gexec is great though for integration tests.

Reply:
With testify I struggled many times to assert on complex objects / object trees. Only Gomega allowed me to do so. On the other hand keeping and learning 2 assert libraries doesn't really make sense for one project. But we can think about it and vote what would be the best option.